docs/getting_started.md

```diff
@@ -20,33 +20,35 @@
 * 编译`configtxgen`工具
 
 	* 如果运行在Linux，在Fabric目录下执行以下命令：
-		
+
 			cd $GOPATH/src/github.com/hyperledger/fabric
 			make configtxgen
 			# 如果出错：'ltdl.h' file not found
 			sudo apt install libtool libltdl-dev
 			# 然后再运行make
 			make configtxgen
-	
+
 	* 如果运行在OSX，先安装Xcode 8.0或以上版本，然后在Fabric目录下执行以下命令：
-	
+
 			# 安装 Homebrew
 			/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 			# 添加 gnu-tar
-			brew install gnu-tar --with-default-names
+			brew install gnu-tar
+			# 将gnu导入到系统环境变量里
+			export PATH="/usr/local/opt/gnu-tar/libexec/gnubin:$PATH"
 			# 添加 libtool
```

Comment:
The reason is '--with-default-names' option removed from Homebrew.
( https://github.com/Homebrew/homebrew-core/pull/35720 )

So the document should be modified to execute Homebrew without that option, then add path of gnutar to the environment variable($PATH) .